docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/migrate-back-end.md

```diff
@@ -0,0 +1,211 @@
+
+To update content-type relations, update the `server/content-types/<content-type-name>/schema.json` file for each content-type with the following procedure:
+
+1. Declare the relation explicitly by setting the `type` attribute value to `"relation"`.
+
+2. Define the type of relation with the `relation` property.<br/>The value should be a string among the following possible options: `"oneToOne"`, `"oneToMany"`, `"manyToOne"` or `"manyToMany"`.
+
+3. Define the content-type target with the `target` property.<br/>The value should be a string following the `api::api-name.content-type-name` or `plugin::plugin-name.content-type-name` syntax convention.
+
+4. (_optional_) In [bidirectional relations](/developer-docs/latest/development/backend-customization/models.md#relations), define `mappedBy` and `inversedBy` properties on each content-type.
+
+::: details Example of all possible relations between an article and an author content-types
+
+  ```json
+  // path: ./src/plugins/my-plugin/server/content-types/article/schema.json
+  
+  // Attributes for the Article content-type
+  "articleHasOneAuthor": {
+    "type": "relation",
+    "relation": "oneToOne",
+    "target": "api::author.author"
+  },
+  "articleHasAndBelongsToOneAuthor": {
+    "type": "relation",
+    "relation": "oneToOne",
+    "target": "api::author.author",
+    "inversedBy": "article"
+  },
+  "articleBelongsToManyAuthors": {
+    "type": "relation",
+    "relation": "oneToMany",
+    "target": "api::author.author",
+    "mappedBy": "article"
+  },
+  "authorHasManyArticles": {
+    "type": "relation",
+    "relation": "manyToOne",
+    "target": "api::author.author",
+    "inversedBy": "articles"
+  },
+  "articlesHasAndBelongsToManyAuthors": {
+    "type": "relation",
+    "relation": "manyToMany",
+    "target": "api::author.author",
+    "inversedBy": "articles"
+  },
+  "articleHasManyAuthors": {
+    "type": "relation",
+    "relation": "oneToMany",
+    "target": "api::author.author"
+  }
+  ```
+
+  ```json
+  // path: ./src/plugins/my-plugin/server/content-types/author/schema.json
+
+  // Attributes for the Author content-type
+  "article": {
+    "type": "relation",
+    "relation": "manyToOne",
+    "target": "api::article.article",
+    "inversedBy": "articleBelongsToManyAuthors"
+  },
+  "articles": {
+    "type": "relation",
+    "relation": "manyToMany",
+    "target": "api::article.article",
+    "inversedBy": "articlesHasAndBelongsToManyAuthors"
+  }
+  ```
+
+:::
+
+## Update plugin configuration
+
+:::callout
+Strapi v3 defines plugin configurations in a `config` folder.
+
+In Strapi v4, the default configuration of a plugin is defined as an object in the `config` property found in the `strapi-server.js` entry file (see [default plugin configuration documentation](/developer-docs/latest/developer-resources/plugin-api-reference/server.md#configuration)).
```

Comment:
Not really sure about this honestly, our CLI doesn't generate new plugins like this:

![image](https://user-images.githubusercontent.com/8593673/150861640-da6e06d5-b960-4aeb-bb3c-7a25fd5633aa.png)

I think we need to decide on a standard and stick to it.

Reply:
Updated in [this commit](https://github.com/strapi/documentation/pull/663/commits/49b142e4ebfb69930b4e94596f19e8d0bfc7dbc7)